components/sysutils/fastfetch/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Carter Li
+#
+
+BUILD_STYLE=cmake
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=           fastfetch
+COMPONENT_VERSION=        2.17.2
+COMPONENT_SUMMARY=        Like neofetch, but much faster because written mostly in C
+COMPONENT_PROJECT_URL=    https://github.com/fastfetch-cli/fastfetch
+COMPONENT_FMRI=           application/fastfetch
+COMPONENT_CLASSIFICATION= Applications/System Utilities
+COMPONENT_SRC=            $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=        $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL=    https://github.com/CarterLi/fastfetch/archive/refs/tags/$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH=   sha256:bb14923615a48fd5372b07a9a922a3540663e07836dd988c7daf95df59f48427
+COMPONENT_LICENSE=        MIT
+COMPONENT_LICENSE_FILE=   LICENSE
+
+TEST_TARGET=$(NO_TESTS)
+include $(WS_MAKE_RULES)/common.mk
+
+CMAKE_OPTIONS += -DSET_TWEAK=Off
+CMAKE_OPTIONS += -DENABLE_LTO=Off
+
+COMPONENT_TEST_CMD = ./fastfetch
```

Comment:
It is strange to have both `TEST_TARGET=$(NO_TESTS)` and `COMPONENT_TEST_CMD = ./fastfetch`.

Reply:
I removed `TEST_TARGET=$(NO_TESTS)` but `COMPONENT_TEST_CMD` didn't run

Reply:
What _exactly_ you tried?  And what _exactly_ was the output?